### ISSUE TITLE
cute_tiled.h 1.2 and loading a tile sheet

### ISSUE THREAD

**Issue by dpeter99** [RandyGaul/cute_headers#114]
Can we expect a 1.2 update of the cute_tiled.h lib?
I already did some work to get it to work with 1.2 files, but i only use a few things from the specification

And can this lib load in a tile sheet file?

**Comment by RandyGaul** [comment]
Support for newest versions will absolutely come. Could you make a PR so I can see your updates? They may help me :)

You can load tilesets that are embedded. Is this what you meant by tile sheet?

**Comment by dpeter99** [comment]
Yeah but tilesets that are in separate files
Will do a PR in a sec

**Comment by RandyGaul** [comment]
I haven’t really considered supporting Tilesets in separate files since it would be a lot of dev work. Since you can embed tilesets into tiled maps, this seemed like a fine solution.

However if someone added tileset loading standalone functions, I would be interested in seeing a pull request.

**Comment by dpeter99** [comment]
Yeah but if the project is bigger than you might want to use separate tilesets.

I don't think i could write it but might give it a try.
